scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title:before{
+    background: url(assets/triangle.svg);
+    background-repeat: no-repeat;
+    content: '';
+    height: $title-decoration-height;
+    position: absolute;
+    right: calc(100% - 20px);
```

Comment:
I thought we said we would use `left: 0;` here to keep the full padding approach